addons/website/static/src/snippets/s_dynamic_snippet_carousel/000.xml

```diff
@@ -11,7 +11,7 @@
         </a>
     </t>
     <t t-name="website.s_dynamic_snippet.carousel">
-        <div t-att-id="uniqueId" class="carousel slide" t-att-data-bs-interval="interval">
+        <div t-att-id="uniqueId" class="carousel slide" t-att-data-bs-interval="interval" data-bs-ride="carousel">
```

Comment:
Is that a "FIX"? "would only start sliding after the first interaction.", you mean that only when you hovered it once, right? If yes, that was not a bug but a choice (between the 2 bootstrap options) and this should be marked as an IMP (change in behavior)?